Version 3/3.5.0/HUSST_Versorgungsdaten_3_5_0.xsd

```diff
@@ -2931,6 +2931,11 @@
     <xs:restriction base="husstDV:INT4">
       <xs:minInclusive value="0"/>
       <xs:maxInclusive value="999"/>
+      <xs:enumeration value="999">

```

Comment:
Kann man so noch die anderen Werte zwischen 1 und 999 konform nutzen? 
Ich kann mir vorstellen, dass die enumeration-Restriktion die min- und maxInclusive überschreiben und damit die anderen Währungen nicht mehr möglich machen.

Reply:
Guter Punkt. 
Sollen wir dann nicht einfach alle Währungen definieren? 
Soo viele sind das ja nicht.

Reply:
Ich meine, wir sollten außerdem auch die 280 - für die DEM noch weiter in unserer Definition mitführen - auch wenn sie keine aktive Währung mehr ist, könnten doch noch einige Archivdaten DEM Beträge enthalten.